nugets/Devolutions.NowProto/INowMessage.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Devolutions.NowProto
+{
+    /// <summary>
+    /// NowMessage definition.
+    /// </summary>
+    public interface INowMessage
+    {
+        /// <summary>
+        /// NowProto message class.
+        /// </summary>
+        static abstract byte TypeMessageClass { get; }
+
+        /// <summary>
+        /// NowProto message kind (class-specific).
+        /// </summary>
+        static abstract byte TypeMessageKind { get; }
+
+        // Workaround for C# not supporting accessing static abstract properties
+        // from descendant interfaces (TypeMessageClass & TypeMessageKind only
+        // useful in generic context)
```

Comment:
question: That’s so weird that we have to do this 🫨 
At least, can you define a default value at the interface level so that it’s not necessary to define both for each implementing class?

Reply:
Yeah, I find that strange and inconvenient too. I also tried to set default values to static `TypeMessageClass`/`TypeMessageClass` values, but again, C# do not allowed me to do so 🤔  Maybe I am just lacking C# knowledge, maybe @thenextman could suggest an alternative/improved solution?